meshtastic/tests/test_node_runtime_channel_export.py

```
@@ -0,0 +1,574 @@
+        0, channel_pb2.Channel.Role.PRIMARY, name="primary", psk=b"\x01"
+    )
+    mock_node.channels = [primary]
+    mock_node.localConfig = localonly_pb2.LocalConfig()
+
+    call_count = {"lora": 0}
+
+    # Create a mock snapshot object that behaves like LocalConfig
+    class MockLocalConfigSnapshot:
+        """Mock LocalConfig that tracks HasField calls for lora."""
+
+        def __init__(self):
```

Ruff (0.15.7)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)